Mathlib/RingTheory/MvPolynomial/MonomialOrder.lean

```diff
@@ -91,6 +117,15 @@ theorem leadingCoeff_zero : m.leadingCoeff (0 : MvPolynomial σ R) = 0 := by
 
 @[deprecated (since := "2025-01-31")] alias lCoeff_zero := leadingCoeff_zero
 
+theorem Monic.ne_zero [Nontrivial R] {f : MvPolynomial σ R} (hf : m.Monic f) :
+    f ≠ 0 := by
+  rintro rfl
+  simp [Monic, leadingCoeff_zero] at hf
+
+theorem Monic.ne_zero_of_ne (h : (0 : R) ≠ 1) {f : MvPolynomial σ R} (hf : m.Monic f) : f ≠ 0 := by
+  nontriviality R
+  exact hf.ne_zero
```

Comment:
What's the point of this lemma? It's a non-idiomatic restatement of `Monic.ne_zero`

Reply:
Idem.